paper.md

```diff
@@ -61,7 +65,10 @@ provides a unified interface for different SBP operators. At the same time,
 the implementations are reasonably fast. Together, this facilitates the development
```

Comment:
this line still states “reasonably fast” without setting this into perspective what it means, comparing it or giving a reference.

Reply:
I added such a perspective later at https://github.com/ranocha/SummationByPartsOperators.jl/pull/109/files#diff-0403ef06adf405f7b310b4518bd6a3559854f54c61676f676ce9cbfee7172ab6R98
(in the section "Features", where I think it might fit better). If you think it should be moved to the section "Statement of need", I will be happy to do so. However, I fear it would interrupt the flow of this paragraph too much if I inserted the paragraph including example timings directly after this sentence.

Reply:
Ah ok.
Yes, my only fear is, that the reasonably fast at this point is still without reference/justification so a reader might be lost where this claim comes from. On the other hand, I also do not like forward references. Maybe at least a short justification (along the lines of “due to multiple dispatch and optimisations per SBP type”) would be nice here?

Reply:
I added such a remark in 7682ffe.

Reply:
This is also ok now.